setup.cfg

```diff
@@ -37,10 +37,10 @@ install_requires =
 dev =
     tox
     black
-    django>=3,<=4
     isort>=5.0.0
 
 test =
+    django>=3,<=4
```

Comment:
What about a new extra ? This way, `pip install procrastinate[django]` would ensure your django version is consistent with other deps ?

Also: `django>=3,<=4` can be written `django~=3.0`

Finally... I'd rather not explicitely remove Django 4 if we have no reason to believe it will fail.

Finally: the current LTS is 2.2, we should be supporting it, I think.

So in the end, I'd say: `django>=2.2`